src/plugins/wb-disable/disable.js

```diff
@@ -53,11 +67,19 @@ var componentName = "wb-disable",
 						localStorage.setItem( "wbdisable", "true" );
 					} catch ( e ) {}
 
-					// Append the Standard version link
-					li.innerHTML = "<a class='wb-sl' href='" + nQuery + "wbdisable=false'>" + i18n( "wb-enable" ) + "</a>";
+					// Temporary fix for i18n until text are translated
+					if ( !noticeHeader && wb.lang === "fr" ) {
+						noticeHeader = i18nTemporary.fr[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.fr[ "disable-notice" ];
+					} else if ( !noticeHeader ) {
+						noticeHeader = i18nTemporary.en[ "disable-notice-h" ];
+						noticeBody = i18nTemporary.en[ "disable-notice" ];
+						noticehtml = noticehtml + " lang='en'";
+					}
```

Comment:
This propose logic will at least nicely fall back on English text if the ```wb.lang``` is not defined in the i18n or in the i18nTemporary. And when the i18n text are falling back to English. the language of the notice section is set to English. That prevent to fail WCAG SC3.1.2 Language of Parts level AA when the current page language is not English or French.